plugins/package-managers/python/src/main/kotlin/Pip.kt

```diff
@@ -130,4 +130,10 @@ class Pip(
             }
         }.getOrThrow()
     }
+
+    private fun detectPythonVersion(workingDir: File): String? {
+        val pythonVersionFile = workingDir.resolve(".python-version")
+        if (!pythonVersionFile.isFile) return null
+        return pythonVersionFile.readText().trim().takeUnless { it.isEmpty() }
```

Comment:
Do you have any reference for the format of the file? Would be nice to link it from a code comment.

Furthermore, would it make sense to make the check a tad more strict? e.g. return null if it's multiline string.

Reply:
> Do you have any reference for the format of the file?

No, see https://github.com/oss-review-toolkit/ort/issues/8366#issuecomment-1972676318.

> Furthermore, would it make sense to make the check a tad more strict? e.g. return null if it's multiline string.

Will do.